apis/kueue/v1alpha1/cohort_types.go

```diff
@@ -60,6 +60,11 @@ type CohortSpec struct {
 	//+listType=atomic
 	//+kubebuilder:validation:MaxItems=16
 	ResourceGroups []kueuebeta.ResourceGroup `json:"resourceGroups,omitempty"`
+
+	// fairSharing defines the properties of the Cohort when
+	// participating in FairSharing.  The values are only relevant
+	// if FairSharing is enabled in the Kueue configuration.
+	FairSharing *kueuebeta.FairSharing `json:"fairSharing,omitempty"`
```

Comment:
NOTE

This matches to design proposal: https://github.com/kubernetes-sigs/kueue/blob/06d4c763ce03891dc0f869e42d5557f9831f1682/keps/1714-fair-sharing/README.md?plain=1#L222-L228